app/utils/kcidb.py

```diff
@@ -176,6 +179,25 @@ def push_tests(group_id, bq_options, db_options={}, db=None):
         'instance': group[models.BOARD_INSTANCE_KEY],
     }
 
+    file_name = "-".join([group[models.NAME_KEY],
+                          group[models.DEVICE_TYPE_KEY]])
+    files = [".".join([file_name, ext]) for ext in ["txt", "html"]]
+
+    lava_json = "-".join(["lava-json", group[models.DEVICE_TYPE_KEY]])
+    lava_json = ".".join([lava_json, "json"])
```

Comment:
The `join()` function is neat with sequences of stings, but in this case it might be better to just use some regular string formatting:
```python
files.append('lava-{}.json'.format(group[models.DEVICE_TYPE_KEY]))
```